### PR TITLE
VM migration code changes

### PR DESCRIPTION
**Pull request by naveena-maplelabs**
1) While creating restore tasks, if the task with same name is completed or cancelled, new task with same name is created.
2) Support to filter tasks based on start and end time is added.
3) By default last one week tasks will be returned. Supported format is DD/MM/YYYY

### CODE REVIEW

File: plugins/module_utils/cohesity_hints.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 import json
 import socket
 import traceback
-
+from datetime import datetime, timedelta
```
Comment (pyashish) [inline plugins/module_utils/cohesity_hints.py:28]: is this in requirements this library ?

Comment (pyashish) [reply plugins/module_utils/cohesity_hints.py:28]: Also I didnt see timedelta used anywhere ?

Comment (naveena-maplelabs) [reply plugins/module_utils/cohesity_hints.py:28]: timedelta is used at line number 587. datetime is inbuilt package.

Comment (pyashish) [reply plugins/module_utils/cohesity_hints.py:28]: Ack, just make sure the package is autoinstalled on upgrade .

**Pull request merged**